samples/boards/nrf52/mesh/onoff_level_app/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _bluetooth-mesh-onoff-sample:
+
+Bluetooth: Mesh OnOff & Level Model
+###########################
+
+Overview
+********
+
+This is a simple application demonstrating a Bluetooth mesh node.
+
+Root element has Generic OnOff Server, Generic OnOff Client, Generic Level Server, Generic Level Client models.
```

Comment:
Start the sentence with:

    The node's root element has ...

and add an ``and`` after the last comma in the sentence:

    ... Generic Level Server, and Generic Level Client models.